projects/plugins/jetpack/extensions/blocks/sharing-button/class-sharing-source-block.php

```diff
@@ -53,6 +53,24 @@ public function get_id() {
 		return $this->id;
 	}
 
+	/**
+	 * Get unique sharing ID. Similar to get_id().
+	 *
+	 * @return int
+	 */
+	public function get_class() {
```

Comment:
The method name is a bit misleading.

Reply:
Yeah, I've copied this one from the original Sharing Sources file, just to be on a safe side. I see it could be gone soon.